doc/sphinxman/source/notes_c.rst

```diff
@@ -60,20 +60,23 @@ Notes on Psivars
 ================
 
 .. note:: Starting in 1.6, there are three standard ways to access an excited state
-    property. We give examples below, but the method name and property name may change.
-    * ``method ROOT 0 -> ROOT m property`` to get root m.
-    * ``method ROOT 0 -> ROOT m property - h TRANSITION`` to get root m and
+   property. We give examples below, but the method name and property name may change.
+   * ``method ROOT 0 -> ROOT m property`` to get root m.
+
+   * ``method ROOT 0 -> ROOT m property - h TRANSITION`` to get root m and
```

Comment:
Should `A2` be replaced with `h` in this sentence? And then maybe mention that a possible value of `h` is `A2`?